packages/manager/src/queries/placementGroups.ts

```diff
@@ -83,7 +83,7 @@ export const useCreatePlacementGroup = () => {
     mutationFn: createPlacementGroup,
     onSuccess: (placementGroup) => {
       queryClient.invalidateQueries(placementGroupQueries.paginated._def);
-      queryClient.invalidateQueries(placementGroupQueries.all._def);
+      queryClient.invalidateQueries(placementGroupQueries.all);
```

Comment:
I think this technically works because the `invalidateQueries` is overloaded so that accepts an object with a `queryKey` OR a string, but I think we should probably do 

```suggestion
      queryClient.invalidateQueries(placementGroupQueries.all.queryKey);
```

Reply:
wait wut? so it reads as invalidating `null`??

Reply:
i know it doesn't, but it feels weird

Reply:
Nope, it will be `["placement-groups", "all"]`!

![Screenshot 2024-03-26 at 12 48 17 PM](https://github.com/linode/manager/assets/115251059/cb93d8db-c5c6-4522-9c47-106185955d9d)

Reply:
yeah i know but it's really unintuitive 😞

Reply:
yeah, I've always been against the explicit `null` in 

```typescript
   all: {
     queryFn: getAllPlacementGroupsRequest,
     queryKey: null,
   },
```

but I've gotten use to it. I'm not sure why `queryKey` is not optional.

Reply:
right.. updated nonetheless, we'll all get used to it 🤷